### ISSUE TITLE
quickMove is only showing last 2 folders in path

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/QuickFolders#172]
Since QuickFolders version 5.6.*

The dropdown menu for quickmove now only shows 3 parts:  account name - toplevel folder » matching folder.  I have many folders named "microsoft", two of which are in account "Ohia mail".  When I use quickmove to move a message to "microsoft", I see a list of several choices:

    Ohia mail - computing » microsoft
    Ohia mail - computing » microsoft
    Mail archive - development » microsoft
    ...

The first two are indistinguishable.  I would used to have seen:

    Ohia mail - ZZ_Last_year » computing » microsoft
    Ohia mail - computing » microsoft
    Mail archive - old_1998 » development » microsoft
    ...

I have the Display Format set to "Account - Folder Path" with Maximum path depth: 6.  None of the other options seem to do what I want, either.l  Before the upgrade, I used to get the longer path names, and I believe the number of path elements was configurable.

What I'd really like to see is Account - path-root » matching-folder.  For the above example, I'd like to see either

    "Ohia mail - ZZ_Last_year » ... » microsoft"
    "Ohia mail - ZZ_Last_year » computing » microsoft"

for the first entry, where "..." represents a number of path elements that exceed a configurable threshold.  A threshold of 0 would result in line (1) above (i.e., don't show any intermediate path elements between the two ends of the path), whereas a threshold of 1 or greater would result in line (2) (i.e., show the middle element of the path).

**Comment by RealRaven2000** [comment]
Only showing 2 items in path only happens if a single word is entered in the search (so neither `>` nor `/` are used). There is a setting extensions.quickfolders.premium.findFolder.maxPathItems which can be set by right-clicking in QuickFolders General / Main Toolbar Elements /  quickMove button, but it also doesn't appear to help here.

**Comment by MichaelJFischer** [comment]
It works as expected now!

**Comment by RealRaven2000** [comment]
> It works as expected now!

Great. Ideally I would like to restrict the number of results  also by using the first folder name entered (if you enter a search string like "grandparent/parent/folder", it might make sense to only show 3 parts of the path rather than the maximum) but it is difficult to determine when this is really helpful - one would have to check all other results to see if there are any duplicates and then increase the length heuristically, which is very difficult to implement as an algorithm.

So I decided to scrap that attempt (for now) and just really stick to the maximum path depth figure in all cases. I also added a point to the change list on the splash screen.

Up to date Tb78 fix (with l10n in change log):
[QuickFolders-mx-5.6.5pre6.zip](https://github.com/RealRaven2000/QuickFolders/files/6764403/QuickFolders-mx-5.6.5pre6.zip)


Thunderbird 68 version with fix: [QuickFolders-wx-4.22.1pre3.zip](https://github.com/RealRaven2000/QuickFolders/files/6764401/QuickFolders-wx-4.22.1pre3.zip)
